src/vivarium_gates_mncnh/data/loader.py

```diff
@@ -577,6 +548,35 @@ def load_no_acs_relative_risk(
     return utilities.set_non_neonnatal_values(no_acs_rr, 1.0)
 
 
+def load_no_acs_paf(
+    key: str, location: str, years: Optional[Union[int, str, List[int]]] = None
+) -> float:
+    p_CPAP_BEmONC = get_data(data_keys.NO_CPAP_RISK.P_CPAP_BEMONC, location, years)
+    p_CPAP_CEmONC = get_data(data_keys.NO_CPAP_RISK.P_CPAP_CEMONC, location, years)
+    rr_no_CPAP = get_data(data_keys.NO_CPAP_RISK.RELATIVE_RISK, location, years)
+    rr_no_ACS = get_data(data_keys.NO_ACS_RISK.RELATIVE_RISK, location, years)
+    p_BEmONC_given_facility = data_values.DELIVERY_FACILITY_TYPE_PROBABILITIES[location][
+        data_values.FACILITY_CHOICE.P_BEmONC
+    ]
+    p_CEmONC_given_facility = 1 - p_BEmONC_given_facility
+    p_BEmONC = (
+        data_values.PROPORTION_IN_FACILITY_DELIVERIES[location] * p_BEmONC_given_facility
+    )
+    p_CEmONC = (
+        data_values.PROPORTION_IN_FACILITY_DELIVERIES[location] * p_CEmONC_given_facility
+    )
+    # marginalize over facility (no CPAP at home)
```

Comment:
it looks like we do have a [variable for CPAP coverage at home](https://github.com/ihmeuw/vivarium_gates_mncnh/blob/86f8e36e8f781549cd6671cba9ad33912638dc4a/src/vivarium_gates_mncnh/constants/data_values.py#L328) that is set to zero. Maybe we should just include that in the equation here so that if that variable is ever changed we won't need to remember to update this equation too?